cli/cmd/install.go

```diff
@@ -63,6 +66,9 @@ func validateAndBuildConfig() (*installConfig, error) {
 		ControllerImage:          fmt.Sprintf("%s/controller:%s", dockerRegistry, conduitVersion),
 		WebImage:                 fmt.Sprintf("%s/web:%s", dockerRegistry, conduitVersion),
 		PrometheusImage:          "prom/prometheus:v2.1.0",
+		GrafanaImage:             "grafana/grafana:5.0.0-beta4",
```

Comment:
I am curious to know what does the `5.0.0-beta4` version have that makes it a viable option over the version `4.6.3`. I don't think this is a big deal at all.

Reply:
Great question! Grafana 5 introduced [datasource provisioning](http://docs.grafana.org/guides/whats-new-in-v5/#data-sources), which allows us to define a data source, like prometheus, via config files:
https://github.com/runconduit/conduit/blob/bf17eef5cb8c3ff188a5be554bc00be1290900ae/cli/cmd/testdata/install_default.golden#L713

Previously, we had to boot Grafana, and then use a separate process to hit Grafana's API to add a datasource:
https://github.com/linkerd/linkerd-viz/blob/master/linkerd-viz#L56